variables.tf

```diff
@@ -54,3 +54,14 @@ variable "delete_default_internet_gateway_routes" {
   description = "If set, ensure that all routes within the network specified whose names begin with 'default-route' and with a next hop of 'default-internet-gateway' are deleted"
   default     = "false"
 }
+
+
+variable "description" {
+  description = "An optional description of this resource. The resource must be recreated to modify this field."
+  default     = ""
+}
+
+variable "auto_create_subnetworks" {
```

Comment:
This should be a boolean I believe.

Reply:
That was my immediate thought.  In main.tf, it is a string
https://github.com/terraform-google-modules/terraform-google-network/blob/4e7cd17b7c7e466fdec245b05d496cd80750a39a/main.tf#L20-L25

Reply:
Historically, strings were auto-converted to booleans but now we can make explicit booleans.